mchf-eclipse/drivers/audio/audio_driver.c

```diff
@@ -3710,7 +3710,7 @@ static void AudioDriver_RxProcessor(AudioSample_t * const src, AudioSample_t * c
             //
             if(dmod_mode != DEMOD_SAM && dmod_mode != DEMOD_AM) // || ads.sam_sideband == 0) // for SAM & one sideband, leave out this processor-intense filter
             {
-                if(ts.filter_path < 48)
+                if(ts.filter_path < 48 && dmod_mode != DEMOD_FM)
```

Comment:
Why only apply this for filters below 48? Reason for asking: Any new or deleted filter (or reordering) would break that condition. So the question is what this the difference for filters lower than 48 compared to the ones above we could use or introduce in the filter description?